packages/wrangler/src/dev/local.tsx

```diff
@@ -83,9 +83,29 @@ function useLocalWorker({
 
       const scriptPath = realpathSync(bundle.path);
 
+      // the wasm_modules/text_blobs/data_blobs bindings are
+      // relative to process.cwd(), but the actual worker bundle
+      // is in the temp output directory; so we rewrite the paths to be absolute,
+      // letting miniflare resolve them correctly
```

Comment:
So in a real "remote" set-up, these files would be uploaded next to the worker, so the relative paths make sense there.
But here, locally, we are not copying the files to the build directory, so we have to refer back to the original source locations.
Right?

Reply:
correct, in the remote setup they get added to the form upload.